### PR TITLE
🧹 Merge operator and webhook binary into a single CLI binary

### PR DESCRIPTION
**Pull request by imilchev**
We have 2 separate binaries now that are build separately - 1 for the operator and 1 for the validating webhook. To implement #378 another binary is going to be needed. Having 3 binaries would make things even more complicated. Since we have just 1 container with both binaries in it, it would make sense to just merge them into a single thing.

To simplify the build process this PR creates 1 CLI using `cobra`. That would allow us to easily extend the operator functionality by just adding commands.

Signed-off-by: Ivan Milchev <ivan@mondoo.com>

### CODE REVIEW

File: pkg/webhooks/handler/webhook.go [hunk 1]
```diff
@@ -72,7 +72,8 @@ func NewWebhookValidator(client client.Client, mode, scanURL, token, integration
 			ApiEndpoint: scanURL,
 			Token:       token,
 		}),
-		clusterID: clusterID,
+		integrationMRN: integrationMRN,
```
Comment (imilchev) [inline pkg/webhooks/handler/webhook.go:75]: This was a bug that I found, which was introduced in #354. The `integrationMRN` was never set, so it was never actually used

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -14,4 +14,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/mondoohq/mondoo-operator
-  newTag: v0.3.1
+  newTag: sha256-f27f50752316d951db39763ba380a3f2de9c8168.sig
```
Comment (joelddiaz) [inline config/manager/kustomization.yaml:17]: Normally, we leave this the same way the `release.sh` script modified it.

File: config/webhook/kustomization.yaml [hunk 3]
```diff
@@ -7,4 +7,4 @@ resources:
 images:
 - name: controller
   newName: ghcr.io/mondoohq/mondoo-operator
-  newTag: v0.0.1
+  newTag: sha256-f27f50752316d951db39763ba380a3f2de9c8168.sig
```
Comment (joelddiaz) [inline config/webhook/kustomization.yaml:10]: Same here.

File: config/manager/kustomization.yaml (hunk 2 above)
Comment (imilchev) [reply config/manager/kustomization.yaml:17]: I overlooked those... will revert them 🥔

**Pull request merged**